getting-started/display.rst

```diff
@@ -93,14 +94,11 @@ Now, use the Butler client to find what data IDs are available for the ``calexp`
 .. code-block:: python
 
    import os
-   collection = f"u/{os.environ['USER']}/single_frame"
-   for ref in butler.registry.queryDatasets('calexp', physical_filter='HSC-R', collections=collection, instrument='HSC'):
+   for ref in butler.registry.queryDatasets('calexp', physical_filter='HSC-R', instrument='HSC'):
        print(ref.dataId)
 
 The printed output are data IDs for the ``calexp`` datasets with the ``HSC-R`` physical filter.
-The ``collections`` and ``instrument`` arguments are both required in this case.
-The first is required because we have not set up default collections to query.
-The second is required because we are filtering on an instrument specific key so we need to say which instrument to use since a butler repository can contain data from multiple instruments.
+The ``instrument`` arguments is required because we are filtering on an instrument specific key so we need to say which instrument to use since a butler repository can contain data from multiple instruments.
```

Comment:
"argument"

Also this is a bit of a run-on sentence.  Maybe:

"Because we wish to filter on an instrument-specific key, but a Butler repository may contain data from multiple instruments, we provide the `instrument` argument to narrow the scope just to the instrument of interest."

Reply:
I've redone that to reflect Jim's comment below about including `instrument` in the butler constructor, so the sentence is now gone.